Test/OrochiUtils.cpp

```diff
@@ -458,7 +462,10 @@ oroFunction OrochiUtils::getFunction( oroDevice device, const char* code, const
 	}
 	oroModule module;
 	oroError ee = oroModuleLoadData( &module, codec.data() );
+
+	oroFunction function;
 	ee = oroModuleGetFunction( &function, module, funcName );
+	OROASSERT( ee == oroSuccess, 0 );
```

Comment:
This will report an error and is detectable in debug mode if the function is not found.